pkg/services/contexthandler/contexthandler.go

```diff
@@ -70,6 +71,11 @@ func (h *ContextHandler) Middleware(c *macaron.Context) {
 		Logger:         log.New("context"),
 	}
 
+	traceID, exist := cw.ExtractTraceID(c.Req.Request.Context())
+	if exist {
```

Comment:
"exists" would be more correct language wise:
```suggestion
	traceID, exists := cw.ExtractTraceID(c.Req.Request.Context())
	if exists {
```